### ISSUE TITLE
Modelica.Blocks.Sources.CombiTimeTable with periodic extrapolation not backward compatible

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1199]
**Reported by mwetter on 3 Jul 2013 22:22 UTC**
The following model works in 3.2, but stops with an error in 3.2.1 beta 3 (Build 0):

```

model TestTable 

    Modelica.Blocks.Sources.CombiTimeTable combiTimeTable(table=[0,1],

      extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic) 

    annotation (Placement(transformation(extent={{-40,0},{-20,20}})));

end TestTable;

```
One can argue that periodic extrapolation of one data point is not defined, but the model worked in 3.2 and hence 3.2.1 is not backward compatible.

A solution is to define periodic extension of one data point to be such that the value is held constant.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1199

**Comment by modelica-trac-importer** [comment]
**Modified by beutlich on 4 Jul 2013 06:55 UTC**

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 4 Jul 2013 08:01 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by beutlich on 4 Jul 2013 11:11 UTC**
Resolved in 3975aaec5ffec83f2e08e21157c15f289a461e88.
